<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="database_name_label">
      <property name="font">
       <font>
        <family>Microsoft JhengHei</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Book Database</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="database_view">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableWidget" name="database_table">
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>238</number>
          </attribute>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Pages</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Cover Type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Category</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="styleSheet">
           <string notr="true">min-height:100px;
</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QPushButton" name="insert_data_button">
             <property name="font">
              <font>
               <family>Microsoft PhagsPa</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">min-height:50px;
max-height:50px;</string>
             </property>
             <property name="text">
              <string>Insert Data</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="data_insert">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Microsoft Sans Serif</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Amount of pages</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="category_input">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">height: 40px;max-width: 200px;
</string>
          </property>
          <item>
           <property name="text">
            <string>Paperback</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Plastic cover</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hardcover</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Leatherbound</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="9" column="1">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="name_input">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">height: 40px;max-width: 200px;
background-color: white;
border: 1px solid gray;
 border-radius: 10px;</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Microsoft Sans Serif</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Cover Type</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Microsoft Sans Serif</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="amount_input">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">height: 40px;max-width: 200px;
</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="10" column="1">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="home_button">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">height: 40px;max-width: 100px;

background-color: white;
border: 1px solid gray;
 border-radius: 10px;</string>
          </property>
          <property name="text">
           <string>home</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Microsoft Sans Serif</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Category</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="cover_type_input">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">height: 40px;max-width: 200px;

background-color: white;
border: 1px solid gray;
 border-radius: 10px;</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="insert_button">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">height: 40px;max-width: 100px;

background-color: white;
border: 1px solid gray;
 border-radius: 10px;</string>
          </property>
          <property name="text">
           <string>Insert</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
